### DISCUSSION TITLE
ESLint rule configuration feedback

### DISCUSSION THREAD

**Discussion by nowyDEV** [nordcloud/eslint-config-pat#16] [General]
Hey, there was a major update to the config recently https://github.com/nordcloud/eslint-config-pat/pull/10

Highlights:
- New rules for consistent naming, sorting, commenting style
- New rules that enhance TypeScript & fill it's gaps (they are gonna be the most useful and also painful to fix)
- New rules for Node.js servers
- New rules to reduce tech debt & force refactoring of a codebase (smaller files, smaller functions, reduced complexity)
- Support for TypeScript 4.9 and `satisfies` operator

Many rules have been added, I'm setting up this thread to discuss them,
If you have any feedback regarding a new config, please feel free to post it here.

Possible topics:
- proposing new rules that would be useful
- complaints about current rules (why we should disable them or change to warnings)
- feedback about lint speed performance

**Comment by matassp** [comment]
The new rules are nice 👍 . 
Just one thing: I saw this rule in eslintrc.js `"unicorn/switch-case-braces"`, but it seems that prettier is cleaning up the braces.

**Reply by nowyDEV** [reply]
Yep, it was misconfiguration when the rules was transitioned to warning, the value inside this config is correct.

**Comment by matassp** [comment]
Would be nice to extend `naming-convention` rule to support `number` type object literal properties:
![image](https://user-images.githubusercontent.com/29577689/208161997-89e9c985-abb1-44c5-a23d-aa5cc7f37041.png)

**Reply by nowyDEV** [reply]
Numbers are now ignored, added in https://github.com/nordcloud/eslint-config-pat/releases/tag/v4.8.0
